apollo-openapi/pom.xml

```diff
@@ -46,6 +46,12 @@
       <artifactId>log4j-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
The scope of this dependency is declared as `provided`, meaning that the dependency won't be included when you include the openapi, if you want to use validation, include the `javax.validation` dependency explicitly. In another word, the dependency included here is to avoid compile time error. If I missed anything, please correct me.

Reply:
@nobodyiam Or can we just create a model class in `AdminService` module and annotated those fields that needs validation there? Then we can remove the dependency in `openapi` module.